lib/rules/no-console.js

```diff
@@ -102,10 +105,31 @@ module.exports = {
         function report(reference) {
             const node = reference.identifier.parent;
 
+            const propertyName = astUtils.getStaticPropertyName(node);
+
             context.report({
                 node,
                 loc: node.loc,
-                messageId: "unexpected"
+                messageId: "unexpected",
+                suggest: [
+                    {
+                        messageId: "removeConsole",
+                        data: { propertyName },
+                        fix(fixer) {
+                            const copyOfStatementListParents = astUtils.STATEMENT_LIST_PARENTS;
+
+                            copyOfStatementListParents.delete("Program");
```

Comment:
This would actually delete `"Program"` from `astUtils.STATEMENT_LIST_PARENTS` because `copyOfStatementListParents` is not really a copy but a reference to the same Set object. This is most likely why some unrelated tests are failing on CI. Anyway, I don't think there's a need to exclude "Program" (see the next comment).

Reply:
Oops, sorry. I misunderstood the previous review. Thanks